content/en/docs/setup/production-environment/container-runtimes.md

```diff
@@ -139,29 +174,32 @@ This section outlines the necessary steps to use containerd as CRI runtime.
 
 Use the following commands to install Containerd on your system:
 
-1. Install and configure prerequisites:
-
-   (these instructions apply to Linux nodes only)
-
-   ```shell
-   cat <<EOF | sudo tee /etc/modules-load.d/containerd.conf
-   overlay
-   br_netfilter
-   EOF
-
-   sudo modprobe overlay
-   sudo modprobe br_netfilter
+#### Install containerd:
 
-   # Setup required sysctl params, these persist across reboots.
-   cat <<EOF | sudo tee /etc/sysctl.d/99-kubernetes-cri.conf
-   net.bridge.bridge-nf-call-iptables  = 1
-   net.ipv4.ip_forward                 = 1
-   net.bridge.bridge-nf-call-ip6tables = 1
-   EOF
+Visit
+[Getting started with containerd](https://containerd.io/docs/getting-started/#starting-containerd)
+and follow the instructions there, up to the point where you have a valid
+configuration file (on Linux: `/etc/containerd/config.toml`).
```

Comment:
` Install containerd: ... Visit` is already present below.

it seems the rebase affected the diff quite a bit.